docs/reference/requirement-sets/excel-api-1-14-requirement-set.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Excel JavaScript API requirement set 1.14
+description: 'Details about the ExcelApi 1.14 requirement set.'
+ms.date: 10/13/2021
+ms.prod: excel
+ms.localizationpriority: medium
+---
+
+# What's new in Excel JavaScript API 1.14
+
+The ExcelApi 1.14 added objects to control the data table feature of a chart, a method to locate all the precedent cells of a formula, and worksheet protection events to track changes to the protection state of a worksheet. It also added multiple `getItemOrNullObject` methods for objects like `CommentCollection`, `ShapeCollection`, and `StyleCollection` to improve error handling.
```

Comment:
Consider linking to the section explaining the `orNullObject` pattern. (Also, please double-check that the link I provided is correct).
```suggestion
The ExcelApi 1.14 added objects to control the data table feature of a chart, a method to locate all the precedent cells of a formula, and worksheet protection events to track changes to the protection state of a worksheet. It also added multiple [`getItemOrNullObject`](../../develop/application-specific-api-model.md#ornullobject-methods-and-properties) methods for objects like `CommentCollection`, `ShapeCollection`, and `StyleCollection` to improve error handling.
```